addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -225,28 +225,15 @@ options.registry.carousel = options.Class.extend({
      * @override
      */
     start: function () {
-        var self = this;
-
         this.$target.carousel({interval: false});
         this.id = this.$target.attr('id');
         this.$inner = this.$target.find('.carousel-inner');
         this.$indicators = this.$target.find('.carousel-indicators');
-        this.$target.carousel('pause');
         this._rebindEvents();
+        this._retargetEdition();
+        this.$target.on('slid.bs.carousel', this._retargetEdition.bind(this));
```

Comment:
I'd write `() => this._retargetEdition()` to avoid giving the event to the method.